### PR TITLE
batches: rename aliases from campaign to batch changes

### LINKED ISSUE #36041: Rename Aliases from Campaign to Batch Changes
In a number of places in the Go code ([`resolver_test.go`](https://sourcegraph.com/github.com/sourcegraph/sourcegraph@rc/credential-check/-/blob/enterprise/cmd/frontend/internal/batches/resolvers/resolver_test.go?L21:2)) we have aliases that reference campaigns. We should rename these to be Batch Changes oriented.

### PR DESCRIPTION
**Pull request by adeola-ak**
Closes #36041
## Test plan
go test

### CODE REVIEW

**Review by Piszmog — APPROVED** [review]

File: enterprise/cmd/frontend/internal/batches/webhooks/gitlab_test.go [hunk 1]
```diff
@@ -580,7 +580,7 @@ func testGitLabWebhook(db *sql.DB) func(*testing.T) {
 				es := createGitLabExternalService(t, ctx, store.ExternalServices())
 
 				// We can induce an error with an incomplete merge request
-				// event that's missing a project.
+				// event that's missing a projebt.
```
Comment (eseliger) [inline enterprise/cmd/frontend/internal/batches/webhooks/gitlab_test.go:583]:
```suggestion
				// event that's missing a project.
```

**Review by eseliger — APPROVED** [review]

File: enterprise/cmd/frontend/internal/batches/resolvers/changeset_counts_test.go [hunk 2]
```diff
@@ -3,7 +3,6 @@ package resolvers
 import (
 	"context"
 	"os"
-	"reflect"
```
Comment (courier-new) [inline enterprise/cmd/frontend/internal/batches/resolvers/changeset_counts_test.go:6]: I'm guessing this got removed after an erroneous _reflect_ -> _reflebt_ change? 😂

File: enterprise/internal/batches/store/batch_specs_test.go [hunk 3]
```diff
@@ -20,7 +20,6 @@ import (
 	"github.com/sourcegraph/sourcegraph/lib/batches/overridable"
 
 	bt "github.com/sourcegraph/sourcegraph/enterprise/internal/batches/testing"
-	ct "github.com/sourcegraph/sourcegraph/enterprise/internal/batches/testing"
```
Comment (courier-new) [inline enterprise/internal/batches/store/batch_specs_test.go:23]: Nice job, whoever originally added `bt` without removing `ct`. 😂

**Review by courier-new — APPROVED** [review]
Woohoo!!! Gotta say, it absolutely made my day seeing the hilarious side ~effebt~ effect of the ct -> bt replacement. 😂 😂 😂 Did another scan and didn't see any other typos, just one dropped import that we might want back!

File: enterprise/internal/batches/store/batch_specs_test.go (hunk 3 above)
Comment (LawnGnome) [reply enterprise/internal/batches/store/batch_specs_test.go:23]: Thank you. Some of my finest work.

**Review by LawnGnome — APPROVED** [review]

**Pull request merged**